server/index.js

```diff
@@ -30,11 +30,6 @@ module.exports = function Server(config, onStart) {
     app.log(['serve', 'exit'], 'Goodbye.');
   });
 
-  // Expose ability to implement middleware via API
-  const use = function use() {
-    app.use.apply(app, arguments);
-  };
-
   // Construct and return mockyeah API
-  return Object.assign({ server }, app.routeManager, { use, config, close });
+  return Object.assign({ server }, app, app.routeManager, { config, close });
```

Comment:
Wasn't sure if we prefer to merge this directly (risk of namespace conflicts), or make it `mockyeah.app`.

We were already exposing `use` from `app`, so I can remove that explicit exposure now, if we go with this way of merging.